.github/workflows/archived/production-pipeline.yml

```
@@ -310,21 +310,21 @@ jobs:
       env:
         ASPNETCORE_ENVIRONMENT: Test
         ConnectionStrings__DefaultConnection: Host=localhost;Database=terrafusion_test;Username=postgres;Password=postgres
-        Redis__ConnectionString: localhost:6379
+        Redis__ConnectionString: localhost:\${{TF_REDIS_PORT:-6379}}
```

actionlint (1.7.7)

got unexpected character ':' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)